ocd-2532.sql

```diff
@@ -0,0 +1,2 @@
+--Remove the column test_functionality.certification_criterion_id_deleted
+ALTER TABLE openchpl.test_functionality DROP COLUMN certification_criterion_id_deleted;
```

Comment:
Does this work if the column is already deleted?

Also should have newline at the end.